rpc/src/v1/types/consensus_status.rs

```diff
@@ -49,8 +49,6 @@ impl Into<ConsensusCapability> for CapState {
 pub enum ReleaseTrack {
 	/// Stable track.
 	Stable,
-	/// Beta track.
-	Beta,
 	/// Nightly track.
 	Nightly,
 	/// Testing track.
```

Comment:
Is there a reason we keep `Testing` here but remove it from `updater/src/types/release_track.rs`? Should we remove it here too?